include/mc_tasks/ImpedanceTask.h

```diff
@@ -196,6 +196,24 @@ struct MC_TASKS_DLLAPI ImpedanceTask : SurfaceTransformTask
     lowPass_.cutoffPeriod(cutoffPeriod);
   }
 
+  /*! \brief Get whether hold mode is enabled. */
+  bool hold() const noexcept
```

Comment:
`inline`

Reply:
Fixed in https://github.com/jrl-umi3218/mc_rtc/pull/143/commits/51d1eb078d2d27fc83085dd1e48545344e8face7